profiles/seccomp/default.json

```diff
@@ -203,7 +208,9 @@
 				"mq_notify",
 				"mq_open",
 				"mq_timedreceive",
+				"mq_timedreceive_time64",
```

Comment:
oh, wow. I copied this from https://github.com/seccomp/libseccomp/commit/bf747eb21e428c2b3ead6ebcca27951b681963a0#diff-a105224843344ca17ce77ae062f22e09R240

so that means it also needs to be fixed in libseccomp?

Oh, I see they switched to a different approach, using  a CSV now? https://github.com/seccomp/libseccomp/blob/4bd8de391d5920bdbdb639b1d81ddd7bfe4d7549/src/syscalls.csv#L213

But still the same

/cc @pcmoore

Reply:
Actually; was it just the commit-message having a typo? https://github.com/torvalds/linux/commit/41ea39101d6b84394fae0c12b702c4326aa71d17

The commit-message says `mq_timedreceiv_time64`, but the code looks to be `mq_timedreceive_time64 `

https://github.com/torvalds/linux/blob/41ea39101d6b84394fae0c12b702c4326aa71d17/arch/arm/tools/syscall.tbl#L435

Reply:
Well, my bad. The version of a patch you refer to in this PR description has it as `mq_timedreceiv_time64`, this was probably a typo fixed in a later incarnation of the same patch.

Reply:
@thaJeztah I'm assuming this was just a misunderstanding and everything is okay now?